dev-python/hatch-docstring-description/hatch-docstring-description-1.1.1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=hatchling
```

Comment:
You might want to add `export SETUPTOOLS_SCM_PRETEND_VERSION=${PV}`, since it seems to use `hatch-vcs` https://github.com/flying-sheep/hatch-docstring-description/blob/5d814288024fdc32f152ddf01e423ce13c2008e7/pyproject.toml#L3

See: https://projects.gentoo.org/python/guide/distutils.html#setuptools-scm-flit-scm-hatch-vcs-pdm-backend-and-snapshots